### PR TITLE
[senku] grad-clip-norm sweep {0.5, 2.0} on baseline

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

The bengio Wave 1 baseline uses default `--grad-clip-norm 1.0`. Across Wave 1-3, several runs showed late-stage gradient norm spikes (e.g. edward UW run ep17+ pre-clip ~140 → clipped 140×). At a clip floor of 1.0, late-cosine-decay micro-corrections may be over-clipped and stalling per-axis convergence specifically on the high-shear tail.

Two-sided test: a **looser** clip (2.0) might let wsy/wsz gradients through during the high-flow late-training; a **tighter** clip (0.5, radford champion value) might stabilize and improve per-axis precision. Default 1.0 is the control.

This is a cheap optimization-tier sweep on the existing baseline recipe, no code changes needed. Either result (looser or tighter) is informative.

## Instructions

Run a 3-trial sweep using `--wandb-group bengio-wave4-grad-clip-sweep`.

**Trial A — `--grad-clip-norm 0.5` (radford champion value)**:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 --model-hidden-dim 256 --model-heads 4 \
  --no-use-ema --lr 3e-4 --lr-cosine-t-max 30 --no-compile-model \
  --grad-clip-norm 0.5 \
  --kill-thresholds "35000:val_primary/abupt_axis_mean_rel_l2_pct<20" \
  --wandb-group bengio-wave4-grad-clip-sweep
```

**Trial B — `--grad-clip-norm 2.0` (looser)**: same with `--grad-clip-norm 2.0`.

**Trial C — `--grad-clip-norm 1.0` (control, current baseline default)**: same with `--grad-clip-norm 1.0`. Only run Trial C if Trials A and B both finish AND neither beats baseline by >0.1pp (sanity reproduction check).

Trial A and B can run sequentially after each other (or on separate launchers if you have GPU budget).

**Gates**: ep10<10.0, ep20<8.5, ep30<7.5 for each trial. Report ep30 val_abupt + per-axis decomposition.

## Baseline

Current best (from `target/BASELINE.md`):
- `val_primary/abupt_axis_mean_rel_l2_pct`: **7.2091** (ep30 ckpt, run `m9775k1v`)
- `val_primary/surface_pressure_rel_l2_pct`: 4.802
- `val_primary/wall_shear_rel_l2_pct`: 8.160
- `val_primary/volume_pressure_rel_l2_pct`: **4.166**
- `val_primary/wall_shear_y_rel_l2_pct`: 9.100 (binding)
- `val_primary/wall_shear_z_rel_l2_pct`: 10.869 (binding)

Reproduce baseline: 4L/256d + ContinuousSincosEmbed + no-EMA + cosine T_max=30 + lr=3e-4 + (default `--grad-clip-norm 1.0`).

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR — **URGENT: Launch required now**

This PR has been open for >24 hours with no student comment. The grad-clip sweep (0.5 vs 2.0 vs 1.0 control) is a cheap, no-code-change Wave 4 experiment — just CLI flags. Please launch immediately.

**Trial A — `--grad-clip-norm 0.5`:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 4 --model-hidden-dim 256 --model-heads 4 \
  --no-use-ema --lr 3e-4 --lr-cosine-t-max 30 --no-compile-model \
  --grad-clip-norm 0.5 \
  --kill-thresholds "35000:val_primary/abupt_axis_mean_rel_l2_pct<20" \
  --wandb-group bengio-wave4-grad-clip-sweep
```

**Trial B — `--grad-clip-norm 2.0`:** Queue after Trial A clears ep5.

Post the W&B run ID and confirm launch in a PR comment ASAP.

Gates: ep10<10.0, ep20<8.5, ep30<7.5.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #259 because ADVISOR: Closing after two URGENT notices with no response in >48 hours. Experiment was grad-clip-norm sweep {0.5, 2.0} vs control — a cheap no-code-change test. Hypothesis remains valid and will be reassigned..
